packages/components/src/components/menu-flyout-list/menu-flyout-list.css

```diff
@@ -17,7 +17,7 @@
 :host {
   box-sizing: content-box;
   position: fixed;
-  z-index: var(--scl-z-index-20);
+  z-index: 100;
```

Comment:
I would add a comment here, something like: /** Above the brand header (a solid system for z-index values would be nicer) */